pkg/lint/rules.go

```diff
@@ -439,6 +439,85 @@ var AllRules = func(l *Linter) Rules { //nolint:gocyclo
 				return fmt.Errorf("no main package or subpackage test found")
 			},
 		},
+		{
+			Name:        "valid-version-stream",
+			Description: "check the provides and tag filter if version stream suffix is passed to package name",
+			Severity:    SeverityError,
```

Comment:
We might want to start with a Warning until we know all existing version streams are fixed up to avoid blocking automated updates.

Reply:
thanks, done - and rebased.